tracing-appender/src/rolling.rs

```diff
@@ -602,7 +602,28 @@ impl Inner {
                     return None;
                 }
 
-                let created = metadata.created().ok()?;
+                let created = metadata.created().ok().or_else(|| {
+                    let datetime = if let (Some(prefix), Some(suffix)) =
+                        (&self.log_filename_prefix, &self.log_filename_suffix)
+                    {
+                        let date = filename
+                            .split(prefix)
```

Comment:
The prefix can be a substring of the suffix.

Reply:
Hmm, correct. I think it's better if remove the complete prefix then suffix from the string.